src/modules/internet/user-agent.ts

```diff
@@ -179,39 +155,29 @@ export function generate(faker: Faker): string {
     ]);
 
   const randomBrowserAndOS = (): [Browser, OS] => {
```

Comment:
While reviewing this PR I saw how we handle the typedefs for inline anonymous functions here which did not occurred to me while reviewing https://github.com/faker-js/faker/commit/09df73aa1ab97dfec4ed3a4dfdd8280f7b094a87#diff-3893a8f03afbc8fb65306679b87c097fb96ee06e745c7bc99f7debd3d280c2e5R80-R181

I would like to change these typedefs to improve readability for humans and want to split TS for JS code a bit by doing so

```suggestion
  const randomBrowserAndOS: () => [Browser, OS] = () => {
```

As an example why this help: "`-`" marks JS while "` `" marks TS
Here you can see that the JS code is splitted multiple times with TS code

```ts
const randomProc = (arch: OS): string =>
------------------------    -        ---

const randomProc: (arch: OS) => string = (arch) =>
----------------                      ------------
```

This gets worse for each additional argument and even more worse if it includes defaults

Maybe later I can search and find a lint rule to enforce that

Reply:
1. Since this rule is not enforced, I'll take your comment as what it is - a suggestion.
2. I personally really hate this kind of declaration. IDK where you get "more readability for humans" in that (even with your explanation)

> Here you can see that the JS code is splitted multiple times with TS code

So what? Could you elaborate on why this is important?

Reply:
Yeah okay, if I find a lint rule to enforce that kind of style I will apply it in its own PR